src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  todoOpener: (todo: Todo) => void;
+};
+
+export const TodoList: React.FC<Props> = ({ todos, todoOpener }) => (
+  <table className="table is-narrow is-fullwidth">
+    <thead>
+      <tr>
+        <th>#</th>
+        <th>
+          <span className="icon">
+            <i className="fas fa-check" />
+          </span>
+        </th>
+        <th>Title</th>
+        <th> </th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {todos.map(todo => (
+        <tr
+          data-cy="todo"
+          className=""
+          key={todo.id}
+        >
+          <td className="is-vcentered">{todo.id}</td>
+          <th className="is-vcentered">
+            <span className="icon">
+              <i className={todo.completed === true
```

Comment:
you can just use convenient utility module named [classnames](https://www.npmjs.com/package/classnames)